### PR TITLE
Added "Home" to the navbar

### PR DESCRIPTION
**Pull request by Vanshiii203**
issue #496 
Screenshot (409)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,6 +68,7 @@
       </div>
       <div class="nav-links" id="navLinks">
         <ul class="links" id="navmenu">
+          <li><a href="#">HOME</a></li>
           <li><a href="#about">ABOUT</a></li>
           <li><a href="#services">SERVICES </a></li>
           <li><a href="html/contact.html">CONTACT US </a></li>
```
Comment (AliReza1083) [inline index.html:74]: There is an extra space, it's better to remove it.
```diff
- <li><a href="#services">SERVICES </a></li>
+ <li><a href="#services">SERVICES</a></li>
```

File: index.html [hunk 2]
```diff
@@ -85,6 +86,7 @@
     </nav>
     <div class="menu" id="menu">
       <ul class="menu-link">
+        <li><a href="#">HOME</a></li>
         <li><a href="#about">ABOUT</a></li>
         <li><a href="#services">SERVICES </a></li>
         <li><a href="html/contact.html">CONTACT US </a></li>
```
Comment (AliReza1083) [inline index.html:92]: There is an extra space, it's better to remove it.
```diff
- <li><a href="#services">SERVICES </a></li>
+ <li><a href="#services">SERVICES</a></li>
```

**Review by AliReza1083 — COMMENTED** [review]
I reviewed your codes, and I came up with this.
Wish you all the best.

**Comment by Susmita-Dey** [comment]
@Vanshiii203 update this

**Review by Susmita-Dey — APPROVED** [review]
LGTM

**Pull request merged**